packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -217,74 +190,77 @@ type State = Readonly<{
  * </MultiSelect>
  * ```
  */
-export default class MultiSelect extends React.Component<Props, State> {
-    labels: Labels;
-
-    static defaultProps: DefaultProps = {
```

Comment:
Default props are set when we deconstruct props in the functional component

https://github.com/Khan/wonder-blocks/blob/e02ada1e89fd30c08b34fabba5658e2547541640/packages/wonder-blocks-dropdown/src/components/multi-select.tsx#L194-L218